docs/source/integrations/pinecone.rst

```diff
@@ -198,7 +204,9 @@ located at `~/.fiftyone/brain_config.json`:
         "similarity_backends": {
             "pinecone": {
                 "api_key": "XXXXXXXXXXXX",
-                "environment": "us-west1-gcp"
+                "cloud": "aws",                 # serverless indexes
+                "region": "us-east-1",          # serverless indexes
+                "environment": "us-east-1-aws"  # pod-based indexes
```

Comment:
Pod-based indexes are not available in the free plan and Pinecone will throw an error when you try to create one. Do you want to include this info in the docs?

Reply:
The default is serverless so I think its fine to assume that if someone specifically chooses pod-based, then they know what they're doing 😄